src/main/java/racingcar/model/domain/Car.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.model.domain;
+
+public class Car {
+
+    private static final int START_POSITION = 0;
+    private static final int MINIMUM_MOVABLE_NUMBER = 4;
+
+    private final String name;
+    private final int position;
+
+
+    private Car(String name) {
+        this.name = name;
+        this.position = START_POSITION;
+    }
+
+
+    private Car(String name, int position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public static Car from(String name) {
+        if (name.length() >= 6 || name.isBlank()) {
+            throw new IllegalArgumentException("한글자 이상, 다섯글자 이하로 입력하세요.");
+        }
+
+        return new Car(name);
+    }
+
+    public int getPosition() {
+        return this.position;
+    }
+
+    public Car moveForward(int number) {
+        if (canMove(number)) {
+            return new Car(this.name, this.position + 1);
+        }
+        return this;
+    }
+
+    private boolean canMove(int number) {
+        if (number < MINIMUM_MOVABLE_NUMBER) {
+            return false;
+        }
+        return true;
```

Comment:
```suggestion
        return number >= MINIMUM_MOVABLE_NUMBER;
```

이렇게 작성되면 더 깔끔할 것 같은데 어떻게 생각하시나용

Reply:
얼리 리턴에 너무 심취해 있어서 이상하게 적용한 것 같습니다🤣. 좋은 의견 감사합니다